src/ivq.tsx

```diff
@@ -492,6 +495,12 @@ export class Ivq extends KalturaPlayer.core.BasePlugin {
     return Array.from(quizQuestionsMap.values());
   };
 
+  private _disablePip = () => {
```

Comment:
please remove to something like `_disableNativePip` to keep difference between dual-screen pip layout and native feature